packages/amplify_flutter/example/integration_test/config_test.dart

```diff
@@ -0,0 +1,282 @@
+import 'dart:io';
```

Comment:
Should this (and all new dart files in this PR) have the big header with copyright info?

Reply:
Fixed